<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapesDialog</class>
 <widget class="QDialog" name="ShapesDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>365</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>OpenSplice|DDS Shapes Demo</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>193</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="wShapeLabel">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Shape:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="colorLabel">
           <property name="text">
            <string>Color:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Size:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Speed:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="wShapeList">
           <item>
            <property name="text">
             <string>Circle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Square</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="colorList">
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Orange</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yellow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Magenta</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cyan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gray</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sizeSlider">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>20</number>
           </property>
           <property name="value">
            <number>45</number>
           </property>
           <property name="sliderPosition">
            <number>45</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="speedSlider">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>20</number>
           </property>
           <property name="value">
            <number>45</number>
           </property>
           <property name="sliderPosition">
            <number>45</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pubQosButton">
       <property name="text">
        <string>QoS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createWriterButton">
       <property name="text">
        <string>Publish</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Shape:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="rShapeList">
         <item>
          <property name="text">
           <string>Circle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Square</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Triangle</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="subQosButton">
       <property name="text">
        <string>QoS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createReaderButton">
       <property name="text">
        <string>Subscribe</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="renderFrame">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>500</width>
     <height>341</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>341</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>341</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>createWriterButton</sender>
   <signal>clicked()</signal>
   <receiver>ShapesDialog</receiver>
   <slot>onPublishButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createReaderButton</sender>
   <signal>clicked()</signal>
   <receiver>ShapesDialog</receiver>
   <slot>onSubscribeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pubQosButton</sender>
   <signal>clicked()</signal>
   <receiver>ShapesDialog</receiver>
   <slot>onWriterQosButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subQosButton</sender>
   <signal>clicked()</signal>
   <receiver>ShapesDialog</receiver>
   <slot>onReaderQosButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPublishButtonClicked()</slot>
  <slot>onSubscribeButtonClicked()</slot>
  <slot>onWriterQosButtonClicked()</slot>
  <slot>onReaderQosButtonClicked()</slot>
 </slots>
</ui>
